pyproject.toml

```
@@ -44,14 +47,17 @@ mkdocs = "*"
 pydocstyle = "*"
 pylint = "*"
 pylint-django = "*"
+python-tss-sdk = ">=1.1.0,<2.0.0"  # (always keep in sync w/ optional dependency)
 requests-mock = "~1.9.3"
 yamllint = "*"
+django-debug = "^0.1.1"
 
 [tool.poetry.extras]
-all = ["boto3", "hvac"]
+all = ["boto3", "hvac", "thycotic"]
 aws = ["boto3"]
 hashicorp = ["hvac"]
 nautobot = ["nautobot"]
+thycotic = ["thycotic"]
```

I believe these should be:

```suggestion
all = ["boto3", "hvac", "python-tss-sdk"]
aws = ["boto3"]
hashicorp = ["hvac"]
nautobot = ["nautobot"]
thycotic = ["python-tss-sdk"]
```